libyul/backends/evm/EVMDialect.h

```diff
@@ -62,25 +62,39 @@ struct BuiltinFunctionForEVM: public BuiltinFunction
  * The main difference is that the builtin functions take an AbstractAssembly for the
  * code generation.
  */
-struct EVMDialect: public Dialect
+struct EVMDialect: Dialect
```

Comment:
Yeah I thought that was weird as well but also didn't want to change around too much. Although when I'm already touching the line, it might as well be made into a class, true. Fixed!